src/routes/safe/store/actions/createTransaction.ts

```diff
@@ -195,7 +230,10 @@ const createTransaction = ({
               state,
             ),
           ])
-          dispatch(fetchTransactions(safeAddress))
+          dispatch({
+            type: 'FETCH_TRANSACTIONS',
+            payload: fetchTransactions(safeAddress),
+          })
```

Comment:
what's up with this syntax? why we couldn't use a previous one?

Reply:
this is more than just a syntax, what you're doing is you're passing a thunk as a payload - this is wrong. We need to improve type safety of our redux setup.